src/app/container/container.component.html

```diff
@@ -26,7 +26,19 @@
     </div>
   </div>
 
-  <div class="row m-1" *ngIf="tool?.mode == 'MANUAL_IMAGE_PATH' && tool?.workflowVersions?.length === 0">
+  <div
+    class="row m-1"
+    *ngIf="
+      (tool?.mode === ModeEnum.AUTODETECTQUAYTAGSAUTOMATEDBUILDS || tool?.mode === ModeEnum.AUTODETECTQUAYTAGSWITHMIXED) && !tool?.gitUrl
+    "
+  >
+    <mat-card class="alert alert-info" role="alert">
+      <mat-icon>warning</mat-icon> This Quay.io tool does not have an associated Git repository. Please view the Quay.io repository and add
+      a build trigger for a valid Git repository.
```

Comment:
Mention they should refresh afterwards?

Reply:
Good thinking, how about
This Quay.io tool does not have an associated Git repository. Please view the Quay.io repository and add a build trigger for a valid Git repository. Once the build has been added, refresh the tool.

Reply:
or

> This Quay.io tool does not have an associated Git repository. Please view the Quay.io repository, add a build trigger for a valid Git repository, and then refresh the tool.

Reply:
I prefer the first one.